mkdocs.yml

```diff
@@ -40,28 +40,36 @@ nav:
   - Cloud:
      - Concepts of cloud computing: cloud/concepts.md
      - Rahti:
-       - Home: cloud/rahti/index.md
-       - Introduction:
-         - Background: cloud/rahti/introduction/background.md
-         # - Technical maturity: cloud/rahti/introduction/technical_maturity.md
-         - Getting access: cloud/rahti/introduction/access.md
-         - Billing: cloud/rahti/introduction/billing.md
-       - Usage:
-         - Getting started: cloud/rahti/usage/getting_started.md
-         - Projects and quota: cloud/rahti/usage/projects_and_quota.md
-         - Command line tool usage: cloud/rahti/usage/cli.md
+       - Contents: cloud/rahti/index.md
+       - What is Rahti: cloud/rahti/rahti-what-is.md
+       - Kubernetes and OpenShift concepts: cloud/rahti/concepts.md
+       - Rahti Tutorials:
```

Comment:
Would it be more logical to have tutorials after the Getting Started section?

Reply:
Yes, I just changed it.